dev/src/index.ts

```diff
@@ -137,6 +137,12 @@ const CLOUD_RESOURCE_HEADER = 'google-cloud-resource-prefix';
  */
 export const MAX_REQUEST_RETRIES = 5;
 
+/*!
+ * The maximum number of times to attempt committing a transaction before
```

Comment:
It's not just committing it, it is retrying the transaction as a whole. Please rephrase.